config/importmap.rb

```diff
@@ -1,3 +1,4 @@
+pin "@ungap/custom-elements", to: "https://ga.jspm.io/npm:@ungap/custom-elements@1.3.0/index.js", preload: true
```

Comment:
The package also supports a `min.js`. Can we use the smaller version, or is it not working?

Reply:
Looks already minified to me https://ga.jspm.io/npm:@ungap/custom-elements@1.3.0/index.js

Reply:
Didn't saw that